packages/node-helper/package.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "@webassemblyjs/node-helper",
+    "version": "1.5.13",
+    "description": "WebAssembly Node Helper",
+    "main": "lib/index.js",
+    "scripts": {
+      "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "author": "Sendil Kumar N",
```

Comment:
The issue is that most of the code in node-helper was actually from other contributors and me. Is that correct?

Reply:
ha ha... `yarn init --yes` 😉 I will change it